app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CategoriesController < ApplicationController
+
+  def show
+    @category = Category.find(params[:id])
+    @videos = @category.videos
+    @series = @category.series
+
+    # @videos.each do |video|
```

Comment:
What's the purpose of this? Anyway, whenever you are going to make http requests, use the `excon` gem ;)

Try to, unless is something you are 100% sure you are going to use soon and have a good reason for not using now, remove comments before sending PRs.

Reply:
Removed, we'll add an image_url field instead of the http requests.